x-pack/plugins/ingest_manager/server/services/epm/kibana/index_pattern/install.ts

```diff
@@ -356,3 +359,35 @@ const getFieldFormatParams = (field: Field): FieldFormatParams => {
   if (field.open_link_in_current_tab) params.openLinkInCurrentTab = field.open_link_in_current_tab;
   return params;
 };
+
+export const ensureDefaultIndices = async (callCluster: CallESAsCurrentUser) =>
+  // create a placeholder index to supress errors in the kibana Dashboards app
```

Comment:
Nit:
```suggestion
  // create a placeholder indicies to supress errors in the kibana Dashboards app
```

Could you also add a link here to the open Kibana issue?